evm/src/libraries/WormholeCctpMessages.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: Apache 2
+pragma solidity ^0.8.22;
+
+import {IWormhole} from "src/interfaces/IWormhole.sol";
+
+import {BytesParsing} from "src/libraries/BytesParsing.sol";
+import {Utils} from "src/libraries/Utils.sol";
+
+library WormholeCctpMessages {
+    using Utils for address;
+    using BytesParsing for bytes;
+
+    // Payload IDs.
+    uint8 private constant DEPOSIT = 0x1;
```

Comment:
Do we want to use a different payload id here? Since technically this isn't an actual token deposit like with the WH bridge?

Reply:
These payload IDs were arbitrarily chosen to start at 1 (uint8), similar to how the Token Bridge's payloads are discriminated against. Because the existing CircleIntegration uses this payload ID to identify its Wormhole messages, we need to keep this payload ID (1).

We now reserve payload IDs 2-10 in case there are more Wormhole messages that could be useful in pairing with CCTP messages.

Reply:
> Because the existing CircleIntegration uses this payload ID to identify its Wormhole messages, we need to keep this payload ID (1)

Great point!

Reply:
Making a note here after talking about this offline: I'll add overloaded methods where an integrator can use his own payload ID (constraining to uint8) as the discriminant for this deposit message.

Reply:
Added here: https://github.com/wormhole-foundation/wormhole-circle-integration/pull/48/commits/916ad81e627cd62be6abd9f0b07b21951704080d